scripts/install_third_party_libs_test.py

```diff
@@ -105,6 +106,16 @@ def mock_ensure_directory_exists(unused_path: str) -> None:
         self.dir_exists_swap = self.swap(
             common, 'ensure_directory_exists', mock_ensure_directory_exists)
 
+    def test_install_third_party_main_under_docker(self) -> None:
+        feconf.OPPIA_IS_DOCKERIZED = True
```

Comment:
Use swap to change config values. (Ditto below)

Reply:
done